client/app/visualizations/word-cloud/Renderer.jsx

```diff
@@ -0,0 +1,187 @@
+import d3 from 'd3';
+import cloud from 'd3-cloud';
+import { each, map, min, max, values, sortBy } from 'lodash';
+import React, { useMemo, useState, useEffect } from 'react';
+import resizeObserver from '@/services/resizeObserver';
+import { RendererPropTypes } from '@/visualizations';
+
+import './renderer.less';
+
+function computeWordFrequencies(rows, column) {
+  const result = {};
+
+  each(rows, (row) => {
+    const wordsList = row[column].toString().split(/\s/g);
```

Comment:
Better to use Lodash's `toString` and `split`  here, otherwise:

![toString](https://user-images.githubusercontent.com/3356951/60366664-8f655a00-99c2-11e9-938e-226c9645af67.png)
(null column values)

This actually happens in the current version as well :point_up_2:

Reply:
@gabrieldutra Done :+1: Thanks!